server.js

```diff
@@ -19,12 +12,89 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+// Defines the routes
+app.get("/", (request, response) => {
+const bookGuide = {
+  Routes: [
```

Comment:
Cool that you wrote a little overview. There's an npm package (express-list-endpoints) that does this for you, if that has any interest.